tests/io/qchem/test_outputs.py

```diff
@@ -264,59 +265,61 @@ class TestQCOutput(PymatgenTest):
+        for file in MULTI_JOB_OUT_NAMES:
+            outputs = QCOutput.multiple_outputs_from_file(f"{TEST_DIR}/{file}", keep_sub_files=False)
             multi_job_dict[file] = [sub_output.data for sub_output in outputs]
         dumpfn(multi_job_dict, "multi_job.json")
 
-    def _test_property(self, key, single_outs, multi_outs):
+    def _check_property(self, key, single_outs, multi_outs):
         for filename, out_data in single_outs.items():
             try:
-                assert out_data.get(key) == single_job_dict[filename].get(key)
+                assert out_data.get(key) == SINGLE_JOB_DICT[filename].get(key)
             except ValueError:
                 try:
                     if isinstance(out_data.get(key), dict):
-                        assert out_data.get(key) == approx(single_job_dict[filename].get(key))
+                        assert out_data.get(key) == approx(SINGLE_JOB_DICT[filename].get(key))
                     else:
-                        assert_allclose(out_data.get(key), single_job_dict[filename].get(key), atol=1e-6)
-                except AssertionError:
-                    raise RuntimeError(f"Issue with {filename=} Exiting...")
-            except AssertionError:
-                raise RuntimeError(f"Issue with {filename=} Exiting...")
+                        assert_allclose(out_data.get(key), SINGLE_JOB_DICT[filename].get(key), atol=1e-6)
+                except AssertionError as exc:
+                    raise RuntimeError(f"Issue with {filename=} Exiting...") from exc
+            except AssertionError as exc:
+                raise RuntimeError(f"Issue with {filename=} Exiting...") from exc
+
         for filename, outputs in multi_outs.items():
             for idx, sub_output in enumerate(outputs):
                 try:
-                    assert sub_output.data.get(key) == multi_job_dict[filename][idx].get(key)
+                    assert sub_output.data.get(key) == MULTI_JOB_DICT[filename][idx].get(key)
                 except ValueError:
                     if isinstance(sub_output.data.get(key), dict):
-                        assert sub_output.data.get(key) == approx(multi_job_dict[filename][idx].get(key))
+                        assert sub_output.data.get(key) == approx(MULTI_JOB_DICT[filename][idx].get(key))
                     else:
-                        assert_allclose(sub_output.data.get(key), multi_job_dict[filename][idx].get(key), atol=1e-6)
+                        assert_allclose(sub_output.data.get(key), MULTI_JOB_DICT[filename][idx].get(key), atol=1e-6)
 
-    @pytest.mark.skip()  # self._test_property(key, single_outs, multi_outs) fails with
-    # ValueError: The truth value of an array with more than one element is ambiguous
+    # PR#3985: the following unit test is failing, and it seems that
+    # the array dimension from out_data and SINGLE_JOB_DICT mismatch
+    @pytest.mark.skip(reason="TODO: need someone to fix this")
```

Comment:
Not sure who is maintaining this module (@samblau maybe? Who appreciate it if you could look like this) `test_all` is failing, and it appears the dimension from `out_data` and `SINGLE_JOB_DICT` mismatch, with the former having one extra dimension for some reason (i.e. `[[a, b, c], ]` vs `[a, b, c]`).